examples/three-validators/hedera-setup.go

```diff
@@ -93,29 +93,48 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
-	fmt.Println("Scheduled Tx Payer Account: ", scheduledTxPayerAccountReceipt.AccountID)
+	fmt.Printf("Scheduled Tx Payer Account: %v\n", scheduledTxPayerAccountReceipt.AccountID)
+	fmt.Println("---Executed Successfully---")
 }
 func cryptoCreate(client *hedera.Client) (hedera.PrivateKey, error) {
 	privateKey, _ := hedera.GeneratePrivateKey()
-	fmt.Println("Hedera Private Key: ", privateKey.String())
+	fmt.Printf("Hedera Private Key: %v\n", privateKey.String())
 	publicKey := privateKey.PublicKey()
 	newAccount, err := hedera.NewAccountCreateTransaction().
 		SetKey(publicKey).
 		SetInitialBalance(hedera.NewHbar(100)).
 		Execute(client)
 	if err != nil {
-		panic(err)
+		return hedera.PrivateKey{}, err
 	}
 	receipt, err := newAccount.GetReceipt(client)
 	if err != nil {
-		panic(err)
+		return hedera.PrivateKey{}, err
+	}
+	fmt.Printf("AccountID: %v\n", receipt.AccountID)
+
+	balance, err := hedera.NewAccountBalanceQuery().
+		SetAccountID(*receipt.AccountID).
+		Execute(client)
+	if err != nil {
+		return hedera.PrivateKey{}, err
 	}
```

Comment:
You don't actually need to query for the balance. You can see it in the `NewAccountCreateTransaction` -> `SetInitialBalance`. This is the line that sent tha HBAR balance of the newly created account